src/Common/SurfaceFluxes/SurfaceFluxes.jl

```diff
@@ -114,10 +114,8 @@ Surface conditions given
  - `x_ave` volume-averaged value for variable `x`
  - `x_s` surface value for variable `x`
  - `z_0` roughness length for variable `x`
- - `F_exchange` flux at the top for variable `x`
  - `VDSE_scale` virtual dry static energy (i.e., basic potential temperature)
- - `Δz` layer thickness (not spatial discretization)
- - `z` coordinate axis
+ - `Δz` layer thickness
```

Comment:
We might want to change the surface fluxes pieces in a separate PR. I can take this on if you'd like

Reply:
Actually, I don't think we'll need a ton of changes